### ISSUE TITLE
[App][Bug]: <Error while extracting multimedia (IMAGES MISSING)>

### ISSUE THREAD

**Issue by cong862023** [sircharlo/meeting-media-manager#2259]
### What happened?

Hello! 
When downloading the media for the next week's Life and Ministry meeting, the image of the Spiritual Treasures was not downloaded. 
Obviously, I downloaded it from the jw.org on my own so that next week's meeting can proceed normally. 
However, I report this bug as it has been recurring. It happened this week, but it has also happened in other weeks. Normally, it always occurs with the image or images of the Tesours Espirituas.
'An additional detail: I have my program and media in Portuguese (Portugal) pt-TPO
Greetings from Portugal

### To Reproduce

1. Open the app
2. Download media when opening automatically
3. Warning appears: "An error occurred while extracting multimedia from the following publication: nwb-20231100"
4. Error: an image of the Spiritual Treasures part is missing.

### What did you expect to happen?

Download all the images of the week.

### Version

v23.10.0

### What platform are you seeing the problem on?

Windows

### Relevant log output

```shell
{
  "1699179334633": [
    "Request failed with status code 404"
  ],
  "1699179461878": [
    "Request failed with status code 404"
  ]
}
```

### Additional context

Windows_NT 10.0.22621 x64
### Anonymized `prefs.json`
```
{
  "app": {
    "autoOpenFolderWhenDone": false,
    "autoQuitWhenDone": false,
    "autoRunAtBoot": false,
    "autoStartSync": true,
    "betaUpdates": false,
    "congregationName": "S. Pedro do Sul",
    "customCachePath": "***",
    "disableAutoUpdate": true,
    "disableHardwareAcceleration": false,
    "localAppLang": "pt-pt",
    "localOutputPath": "***",
    "obs": {
      "enable": true,
      "port": "4444",
      "password": "***",
      "mediaScene": "1-VÍDEOS_E_IMAGENS",
      "imageScene": null,
      "cameraScene": "2-CÂMERA_ORADOR",
      "zoomScene": null,
      "useV4": true
    },
    "offline": false,
    "outputFolderDateFormat": "DD-MM-YYYY - dddd",
    "theme": "system",
    "zoom": {
      "enable": false,
      "name": null,
      "id": null,
      "password": null,
      "spotlight": false,
      "hideComponent": false,
      "autoRename": "***",
      "autoStartMeeting": false,
      "autoStartTime": 1
    }
  },
  "cong": {
    "server": null,
    "port": null,
    "user": null,
    "password": null,
    "dir": null
  },
  "media": {
    "autoPlayFirst": false,
    "autoPlayFirstTime": 5,
    "enableMediaDisplayButton": true,
    "enableMp4Conversion": false,
    "enablePp": false,
    "enableSubtitles": true,
    "enableVlcPlaylistCreation": false,
    "excludeLffImages": false,
    "excludeTh": true,
    "excludeFootnotes": false,
    "hideMediaLogo": false,
    "hideWinAfterMedia": false,
    "includePrinted": false,
    "keepOriginalsAfterConversion": false,
    "lang": "TPO",
    "langFallback": null,
    "langSubs": "TPO",
    "langUpdatedLast": "2023-08-10T20:23:40.842Z",
    "maxRes": "720p",
    "mediaWinShortcut": "Alt+Z",
    "ppBackward": null,
    "ppForward": null,
    "preferredOutput": 2779098405,
    "presentShortcut": "Alt+D"
  },
  "meeting": {
    "autoStartMusic": true,
    "enableMusicButton": true,
    "enableMusicFadeOut": true,
    "coWeek": null,
    "musicFadeOutTime": 60,
    "musicFadeOutType": "smart",
    "musicVolume": 100,
    "mwDay": 3,
    "mwStartTime": "19:30",
    "shuffleShortcut": "Alt+K",
    "specialCong": false,
    "weDay": 5,
    "weStartTime": "19:00"
  },
  "__internal__": {
    "migrations": {
      "version": "23.10.0"
    }
  }
}
```
### Error details
```
{
  "stack": "Error: ENOENT: no such file or directory, stat 'C:/Users/Utilizador/Videos/MEDIA_REUNIÕES/Publications/T/mwb/20231100/0/202023402_univ_cnt_1.jpg'\n    at statSync (node:fs:1690:3)\n    at Object.<anonymous> (node:electron/js2c/asar_bundle:2:4807)\n    at app://./_nuxt/13d49b4.js:5:783438\n    at app://./_nuxt/623a80f.js:1:575207\n    at app://./_nuxt/623a80f.js:1:575303\n    at Array.forEach (<anonymous>)\n    at W (app://./_nuxt/623a80f.js:1:573076)\n    at H (app://./_nuxt/623a80f.js:1:582487)\n    at app://./_nuxt/623a80f.js:1:584382\n    at Array.forEach (<anonymous>)\n    at f.<anonymous> (app://./_nuxt/623a80f.js:1:584362)\n    at async f.getMwMedia (app://./_nuxt/73441ba.js:1:22657)\n    at async Promise.allSettled (index 0)\n    at async f.startMediaSync (app://./_nuxt/73441ba.js:1:25372)",
  "message": "ENOENT: no such file or directory, stat 'C:/Users/Utilizador/Videos/MEDIA_REUNIÕES/Publications/T/mwb/20231100/0/202023402_univ_cnt_1.jpg'",
  "errno": -4058,
  "syscall": "stat",
  "code": "ENOENT",
  "path": "C:/Users/Utilizador/Videos/MEDIA_REUNIÕES/Publications/T/mwb/20231100/0/202023402_univ_cnt_1.jpg"
}
```

**Comment by mtdvlpr** [comment]
Hi @cong862023, thanks for reporting! It's been fixed in #2260 and the fix will be released shortly. It was a rare edge case that caused the location of the image to be incorrectly calculated

**Comment by cong862023** [comment]
> Hi @cong862023, thanks for reporting! It's been fixed in #2260 and the fix will be released shortly. It was a rare edge case that caused the location of the image to be incorrectly calculated

Thank you
